firefox-ios/Client/Frontend/Autofill/Address/AddressCellView.swift

```diff
@@ -0,0 +1,84 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import SwiftUI
+import Common
+import Shared
+import Storage
+
+// MARK: - AddressCellView
+
+/// A view representing a cell displaying address information.
+struct AddressCellView: View {
+    // MARK: - Properties
+
+    @State private var textColor: Color = .clear
+    @State private var customLightGray: Color = .clear
+    private(set) var address: Address
+    @Environment(\.themeType)
+    var themeVal
+    private(set) var onTap: () -> Void
+
+    // MARK: - Body
+
+    var body: some View {
+        Button(action: onTap) {
+            VStack(alignment: .leading, spacing: 0) {
+                HStack(spacing: 24) {
+                    Image(ImageIdentifiers.location)
+                        .padding(.leading, 16)
+                        .foregroundColor(.primary)
+                        .offset(y: -14)
+                    VStack(alignment: .leading) {
+                        Text(address.givenName + " " + address.familyName)
+                            .preferredBodyFont(size: 17)
```

Comment:
instead of a hardcoded size, can we follow our typography system? https://www.figma.com/file/pEyGeE4KV5ytYHeXMfLcEr/Mobile-Styles?type=design&node-id=501-2879&mode=design&t=w14NZV7XIpVr6uCm-0

The figma file should be using fonts from the system, if not we should discuss with designs to follow the design system.
Example screenshot from Figma file:
![image](https://github.com/mozilla-mobile/firefox-ios/assets/6743397/288fbb9e-40d8-4926-8234-70bdfa42094d)

Reply:
changed